scripts/BigBrain_crawler.py

```diff
@@ -0,0 +1,31 @@
+import os
+from ftplib import FTP
+
+
+host = 'bigbrain.loris.ca'
+directory = 'BigBrainRelease.2015/3D_Surfaces/Apr7_2016'
```

Comment:
@emmetaobrien does that mean that for each BigBrain dataset, that needs to be changed? Maybe adding some documentation on how to use the script would be useful?